main.tex

```diff
@@ -777,13 +818,9 @@
 
 \end{enumerate}
 
-\item  An Owner or Occupier of a Lot without the consent of the Owners Corporation must not move any bulky goods or heavy goods, into or out of the Development except between the hours of 9.00AM and 5.00PM on Business Days and then only when the lift protection covers have been installed.
```

Comment:
This time is inconsistent with https://github.com/owen9825/apartment-rules/pull/1/files#diff-f67098301b6788c19af2853a0e5ad776fda05572b1b1224df46c42d1ad323cbbL1544

The new time I've written here is what was already written in this other section.